src/Video/Lighting/Lighting.cpp

```diff
@@ -54,57 +55,53 @@ void Lighting::AddLight(const Light& light) {
 }
 
 void Lighting::Render(const glm::mat4& inverseProjectionMatrix, RenderSurface* renderSurface) {
-    // Disable depth testing
-   // GLboolean depthTest = glIsEnabled(GL_DEPTH_TEST);
-   // glEnable(GL_DEPTH_TEST);
-   // 
-   // GLint oldDepthFunctionMode;
-   // glGetIntegerv(GL_DEPTH_FUNC, &oldDepthFunctionMode);
-   // glDepthFunc(GL_ALWAYS);
-    
-    // Blending enabled for handling multiple light sources
-   // GLboolean blend = glIsEnabledi(GL_BLEND, 0);
-   // glEnablei(GL_BLEND, 0);
-   // glBlendEquationi(0, GL_FUNC_ADD);
-   // glBlendFunci(0, GL_ONE, GL_ONE);
-    
-  //  shaderProgram->Use();
-    
-    
+    //// Disable depth testing.
+    //GLboolean depthTest = glIsEnabled(GL_DEPTH_TEST);
+    //glEnable(GL_DEPTH_TEST);
+    // 
+    //GLint oldDepthFunctionMode;
+    //glGetIntegerv(GL_DEPTH_FUNC, &oldDepthFunctionMode);
+    //glDepthFunc(GL_ALWAYS);
+    //
+    //// Blending enabled for handling multiple light sources.
+    //GLboolean blend = glIsEnabledi(GL_BLEND, 0);
+    //glEnablei(GL_BLEND, 0);
+    //glBlendEquationi(0, GL_FUNC_ADD);
+    //glBlendFunci(0, GL_ONE, GL_ONE);
+    //
+    //shaderProgram->Use();
+    //
+    //
     //glBindVertexArray(rectangle->GetVertexArray());
-    
-    // Set uniforms.
+    //
+    //// Set uniforms.
     //glUniform1i(shaderProgram->GetUniformLocation("textureAlbedo"), 0);
     //glUniform1i(shaderProgram->GetUniformLocation("textureNormal"), 1);
     //glUniform1i(shaderProgram->GetUniformLocation("textureMetallic"), 2);
     //glUniform1i(shaderProgram->GetUniformLocation("textureRougness"), 3);
     //glUniform1i(shaderProgram->GetUniformLocation("tDepth"), 4);
-    //
+    
     glUniform1i(shaderProgram->GetUniformLocation("lightCount"), lightCount);
     glUniformMatrix4fv(shaderProgram->GetUniformLocation("inverseProjectionMatrix"), 1, GL_FALSE, &inverseProjectionMatrix[0][0]);
+    
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') The scope of the variable 'lightIndex' can be reduced. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/coding_rules#rule_key=cppcheck%3AvariableScope)